jumptrading/aminal
# Fix -ignore-config flag behavior

## Description

Previously, the `-ignore-config` command-line flag was not working. Now it works according to the following logic:

- If the `-ignore-config` is specified then the built-in configuration defaults are used, else Aminal tries to load configuration file.
- The parameters `-shell`, `-debug`, and `-slomo` are actually overriding corresponding values specified in the configuration (whether default or loaded from a configuration file).

Fixes #8 

## Type of change


## How Has This Been Tested?

- Run Aminal without parameters to ensure the default configuration file is created, and then close.
- Edit `%USERPROFILE%\.config\aminal\config.toml` and change `shell` parameter to `"cmd.exe /K echo test"`.
- Run Aminal without parameters again and see the `test` text at the top of the Aminal window. That proves that it uses the shell specified in the configuration file. Close Aminal.
- Run Aminal with `-shell cmd.exe` parameter. Now there will be no `test` message. This proves that the `shell` configuration parameter has been overridden. Close Aminal.
- Run Aminal with `-ignore-config` parameter. Again, there will be no `test` message. This proves that the entire configuration file has been ignored and the built-in default configuration has been used.

**Test Configuration**:
* OS: `Windows`
* OS version: `10`
* Go version: `go1.11.2 windows/amd64`
